package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nameless",
+  "version": "2.0.0-pr13",
+  "repository": "https://github.com/NamelessMC/Nameless",
+  "license": "MIT",
```

Comment:
Can we add `"private": true,` to here so it's not possible to accidentally publish this to npm